docs/tutorials/hello-tye/postgres.yaml

```diff
@@ -0,0 +1,47 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: postgres
+  labels:
+    app.kubernetes.io/name: postgres
+    app.kubernetes.io/part-of: microservice
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: postgres
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: postgres
+        app.kubernetes.io/part-of: microservice
+    spec:
+      containers:
+        - name: postgres
+          image: postgres
+          resources:
+            requests:
+              cpu: 100m
```

Comment:
Why these limits specifically?

Reply:
This is to set requested configuration for the container. That's the reason it has been set as `request` not as `limit`. This is also to keep it synced with `redis.yml`. Do you want me to remove that ? Thoughts ?

Reply:
I would personally just remove these limits.

Reply:
Removed the resource limits from the `postgres.yaml`